internal/runtime/hcsv2/workload_container.go

```diff
@@ -171,6 +176,22 @@ func setupWorkloadContainerSpec(ctx context.Context, sbid, id string, spec *oci.
 	return nil
 }
 
+// Helper function to create an oci prestart hook to run ldconfig
+func addLDConfigHook(ctx context.Context, spec *oci.Spec, args, env []string) error {
```

Comment:
doesn't look like this can return an error. is it just to have similar behavior as other hooks?